ote/src/cljs/ote/app/controller/own_services.cljs

```diff
@@ -83,4 +84,14 @@
                        (url-util/encode-url-component transport-operator-id))
                   {}
                   {:on-success (tuck/send-async! ->RemoveSelectionSuccess transport-operator-id service-id)})
-                app))
\ No newline at end of file
+                app))
+
+; Ensure that :transport-operator is not nil
```

Comment:
```suggestion
;; Ensure that :transport-operator is not nil
```

Reply:
Could this comment what case is this check fixing? A bit surprising to see `app` can be nil.

Reply:
Instead of `VerifyOperatorSelection` should the event be `InitOwnServices` or something similar?
If we had an event naming convention, it could be "Init<view or component name>"